src/ResourceManager/RedisCache/Commands.RedisCache/AzureRM.RedisCache.psd1

```diff
@@ -55,7 +55,9 @@ RequiredModules = @(@{ModuleName = 'AzureRM.Profile'; ModuleVersion = '5.5.1'; }
 
 # Assemblies that must be loaded prior to importing this module
 RequiredAssemblies = '.\Microsoft.Azure.Insights.dll', 
-               '.\Microsoft.Azure.Management.Redis.dll'
+               '.\Microsoft.Azure.Management.Redis.dll',
+               '.\Microsoft.Data.Edm.dll', '.\Microsoft.Data.OData.dll', 
+               '.\Microsoft.Data.Services.Client.dll', '.\System.Spatial.dll'
```

Comment:
This PR should not have any changes for RedisCache. These files are no longer needed for Redis. Have you merged latest from the preview branch?

Reply:
Reverted